code/modules/organs/internal/appendix.dm

```diff
@@ -40,5 +40,5 @@
 				E.sever_artery()
 				E.germ_level = max(INFECTION_LEVEL_TWO, E.germ_level)
 				owner.adjustToxLoss(25)
-				removed()
+				uninstall()
```

Comment:
Same here, though harder to tell with the qdel on the next line.

Reply:
For this one, we don't know or care that we're on a mob, or inside a body part. (Even though, that probably shouldn't ever happen when inside a body part..)
And qdel already handles clearing parent refs. I guess, I don't even really need to call uninstall there.

Reply:
Removed the call to uninstall here, and let qdel handle it.